Configurations.md

```diff
@@ -926,6 +926,14 @@ fn add_one(x: i32) -> i32 {
 }
 ```
 
+## `doc_comment_code_block_width`
+
+Max width for code snippets included in doc comments. Only used if `format_code_in_doc_comments` is true.
+
+- **Default value**: `100`
+- **Possible values**: any positive integer that is less than or equal to the value specified for [`max_width`](#max_width)
+- **Stable**: No (tracking issue: [#5359](https://github.com/rust-lang/rustfmt/issues/5359))
+
```

Comment:
I think it would be good to include a link to the `format_code_in_doc_comments` option, similar to what's done for other options https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#array_width

What do you think?